### PR TITLE
add serde feature for rgb/serde

### PR DESCRIPTION
**Pull request by LuckyTurtleDev**

### CODE REVIEW

**Comment by msrd0** [comment]
I don't see why this is necessary? `lottieconv` library has nothing to do with serde

File: lottieconv/Cargo.toml [hunk 1]
```diff
@@ -30,6 +30,7 @@ required-features = ["clap", "webp"]
 default = ["gif", "webp"]
 gif = ["gif_crate"]
 webp = ["webp-animation"]
+serde = ["rgb/serde"]
```
Comment (msrd0) [inline lottieconv/Cargo.toml:33]: Please keep alphabetic ordering of the features (except `default`, that should always be on top)

**Review by msrd0 — CHANGES_REQUESTED** [review]

**Comment by LuckyTurtleDev** [comment]
> I don't see why this is necessary? lottieconv library has nothing to do with serde

I would like do deserialize the reexported color type from a json.

**Comment by msrd0** [comment]
Hm ok I see, `rgb` is being re-exported so you don't necessarily have that in your cargo.toml file. However, I don't see how this is specific to the `lottieconv` crate. If we add this feature, it should be added to `rlottie` itself and then only passed through by `lottieconv`.

File: lottieconv/Cargo.toml [hunk 2]
```diff
@@ -29,6 +29,7 @@ required-features = ["clap", "webp"]
 [features]
 default = ["gif", "webp"]
 gif = ["gif_crate"]
+serde = ["rgb/serde"]
```
Comment (msrd0) [inline lottieconv/Cargo.toml:32]: This should enable `rlottie/serde`

**Review by msrd0 — CHANGES_REQUESTED** [review]

**Review by msrd0 — APPROVED** [review]

**Pull request merged**